test/testSamples/MaterialX/color_management_USD.usda

```diff
@@ -50,6 +50,7 @@ def Mesh "pPlane1" (
                 uniform token info:id = "UsdUVTexture"
                 float4 inputs:fallback = (0.5, 0.5, 0.5, 1)
                 asset inputs:file = @textures/color_palette_ACEScg.exr@
+                token inputs:sourceColorSpace = "ACEScg"
```

Comment:
Technically incorrect since the UsdUVTexture spec has only 3 options (raw/sRGB/auto). But MayaUSD will handle those.

Tried adding that as metadata, but it seems it only works on the MaterialX side.